_topics/en/soft-fork-activation.md

```diff
@@ -0,0 +1,603 @@
+  proposal after vocal opposition from a large number of individual
+  users and several businesses.
+
+  It remains a subject of debate exactly how much each of the above
+  events, and other events occurring around the same time, actually
+  contributed to segwit being activated.
+
+  ### Emergency activations
+
+  In several cases where a serious vulnerability was found in consensus
+  code, developers released a fix without any activation conditions.
+  This risked a consensus failure but it also eliminated the immediate
+  vulnerability for each node as soon as it upgraded.  Notable
+  occasions included:
+
+  - **Replace height with chainwork (July 2010):** Bitcoin was originally
+    implemented to follow the valid chain with the most blocks ("longest
+    chain").  If every block had the same difficulty, that would be the
+    chain with the most Proof-of-Work (PoW).  But different blocks can
+    have different difficulty, so the [chainwork][] soft fork was
+    released in [Bitcoin 0.3.3][] to follow the valid chain with the
+    greatest cumulative proof of work.
+
+  - **Eliminate script bypass bug (July 2010):** Bitcoin originally
+    combined the script that spent a UTXO (the scriptSig) with the
+    script that secured that UTXO (the scriptPubKey) and evaluated them
+    at the same time.  This could allow someone to cause the script to
+    terminate in success before its locking mechanism had been evaluated
+    (e.g. before running `OP_CHECKSIG` to verify a signature was valid).
+    The bug was originally [reported][1 OP_RETURN] as a scriptSig of
+    `OP_TRUE OP_RETURN` allowing spending anyone's bitcoins.  Initially
+    this was [fixed][op_return always false] in [Bitcoin 0.3.6][] by
+    causing the `OP_RETURN` opcode to always result in failure and by
+    placing a number of other limits on scripts.  Although all of those
+    changes were soft fork changes, the same code revision also made a
+    hard forking [change][op_nopx] that would allow removing some of
+    those limitations later.  Despite those significant changes, the
+    underlying problem of scriptSigs being able to tamper with the
+    execution of scriptPubKeys remained and so a [second soft
+    fork][split evalscript] implemented in [Bitcoin 0.3.8][] began
+    evaluating scriptSigs independently of scriptPubKeys. <!-- technically
+    implemented in 0.3.7 according to revision history, but there's no
+    release announcement for that which I can find. -->
+
+  - **Fix value overflow bug (August 2010):** someone
+    created a transaction that spent 0.5 BTC to two outputs worth
+    92,233,720,368.54277039 BTC.  Bitcoin requires that the output
+    amount be less than or equal to the input amount, but this was
+    checked by adding the output values into a single 64-bit
+    floating-point value which can only hold a maximum value of
+    92,233,720,368.54776 before rolling over to negative numbers,
```

Comment:
Should this be 92,233,720,368.54775807 BTC / 9,223,372,036,854,775,807 sats (the largest possible signed 64-bit integer)?